LinuxLogParsers/LinuxLogParser/AndroidLogcat/AndroidLogcatLogParser.cs

```diff
@@ -79,10 +81,28 @@ public AndroidLogcatLogParser(string[] filePaths) : base(filePaths)
 
                     var androidLogCatMatch = AndroidLogCatRegex.Match(line);
 
+                    if (androidLogCatMatch.Success && timeDateFormat == null)
+                    {
+                        const string monthDayFormat = "MM-dd HH:mm:ss.fff";
+                        const string monthDayYearFormat = "MM-dd-yyyy HH:mm:ss.fff";
+                        if (DateTime.TryParseExact(androidLogCatMatch.Groups[1].Value, monthDayFormat, dateTimeCultureInfo, DateTimeStyles.None, out DateTime parsedTimeTmp1))
```

Comment:
nit: Use "out _" for unused out variable

Reply:
fixed